src/main/java/us/monoid/web/JSONResource.java

```diff
@@ -34,6 +34,13 @@ public JSONArray array() throws IOException, JSONException {
 		return (JSONArray)json;
 	}
 
+	/**
+	/* Parse and return a boolean value
+	/*
+	public boolean bool() throws IOException, JSONException {
+		return (boolean)unmarshal();
```

Comment:
I think unmarshal() should only be called if it hasn't been unmarshalled already, and the value cached in ```json``` be cast and returned

```java
if (json == null) unmarshal();
return (boolean)json;
```